.env

```diff
@@ -27,6 +27,6 @@ APP_SECRET=c1cdcd0a21a1ffeee4c1dc26b5fe228b
 # IMPORTANT: DATABASE_URL *must* define the server version
 #
 # DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
-DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=8.0"
+#DATABASE_URL="mysql://TarikElha:J-e/c*51ode@127.0.0.1:3306/couette?serverVersion=8.0"
```

Comment:
remove it, your db logs should not appears in github ! 
Copy your env file and paste it, rename it by .env.local and add your logs on it